CMakeLists.txt

```diff
@@ -1,6 +1,10 @@
 # Verify minimum required version
 cmake_minimum_required(VERSION 2.8.12)
 
+if(CMAKE_VERSION VERSION_EQUAL 3.1 OR CMAKE_VERSION VERSION_GREATER 3.1)
```

Comment:
Why is this needed? We already set the -std=c++11 flag here: https://github.com/dotnet/coreclr/blob/master/src/pal/tools/clang-compiler-override.txt

Reply:
OK. I have been using a hacked up cmake build environment for compiling the PAL layer only. I'll drop this change.